content/troubleshooting/setup/faq/Libraries-FAQ.md

```diff
@@ -0,0 +1,32 @@
+<!--
+title: "Libraries"
+description: "Library recognition issues"
+tags: "troubleshoot setup FAQ library"
+-->
+
+## Why does my open-source library show up as unknown?
+
+Contrast identifies libraries by their SHA-1 digest, and updates library definitions periodically. As a result, Contrast might not recognize new libraries when agents report them. 
+
+> **Note:** If any library repackaging occurs for **Java** clients, which WebSphere does by default, the digest is different. To prevent repackaging, you can add the following JVM system property:
+```
+-Dorg.eclipse.jst.j2ee.commonarchivecore.ignore.web.fragment=true
+``` 
+
+## Why doesn't Contrast recognize my library as the most-recent version?
+
+Issues with library version recognition result from the way that Contrast data sources store information about a library's version. Some of the versions are formatted to include the year at the beginning, and so the sort method reads them as a more recent version. The problem should phase itself out as the industry is moving to a more normalized version convention.
```

Comment:
This is Java/maven specific.  .NET uses Nuget and Node uses npm.  Our reasons for not finding a library a slightly different.

Reply:
@ldemidov Will add a note here, and plan to address this further in the future branch/PR  (which will be coming soon!)